Dockerfiles/midstream/unit_tests.py

```diff
@@ -0,0 +1,59 @@
+
+import unittest
+import os
+import subprocess
+
+
+class Testing(unittest.TestCase):
+    # Running the container with correctly set environment variables, all tasks from run_tests.py should pass
+    def test_postive(self):
+        exec_cmd = """
+        /run_tests.py
+        """
+        result = subprocess.run(exec_cmd, shell=True)
+        self.assertEqual(0, result.returncode)
+
+    # Set incorrect env variable UMOCI_BIN_PATH for ansible playbook extract-operator-bundle.yml - playbook fails to extract
+    def test_negative_extract_operator_bundle(self):
+        exec_cmd = """
+        export UMOCI_BIN_PATH=/root
+        /run_tests.py
+        """
+        result = subprocess.run(exec_cmd, shell=True)
+        with open("extract-operator-bundle.errormessage") as error_file:
+            if "Result code:" not in error_file.read():
```

Comment:
Can we change if to assert ? (self.assert.*)

Reply:
Will use self.assertTrue(), I would say it is the best fit here.

Reply:
There is also `assertIn`
https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertIn

Reply:
Desn't improve readability or anything in code in my opinion, but implemented.